server/fishtest/views.py

```diff
@@ -1370,15 +1370,25 @@ def tests_user(request):
     username = request.matchdict.get("username", "")
     response = {**get_paginated_finished_runs(request), "username": username}
     if int(request.params.get("page", 1)) == 1:
-        response["runs"] = request.rundb.aggregate_unfinished_runs(username)[0]
+        response["runs"] = request.rundb.aggregate_unfinished_runs(
+            username=username,
+        )[0]
     # page 2 and beyond only show finished test results
     return response
 
 
 def homepage_results(request):
     # Calculate games_per_minute from current machines
     games_per_minute = 0.0
-    machines = request.rundb.get_machines()
+    unfinished_runs = request.rundb.get_unfinished_runs()
```

Comment:
spurious line: `unfinished_runs = request.rundb.get_unfinished_runs()`

Reply:
Sorry about that!